### PR TITLE
DigitalOcean: new module: digital_ocean_account_facts

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py

##### ANSIBLE VERSION

```
2.6devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
@pmarques Could you please review this ?

**Comment by Akasurde** [comment]
Closing and re-opening to trigger CI.

File: lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    oauth_token: "{{ oauth_token }}"
+'''
+
+
+RETURN = '''
+data:
+    description: DigitalOcean account facts
+    returned: success
+    type: dictionary
+    sample: {
```
Comment (BondAnthony) [inline lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py:46]: Change sample to `data` that will represent what is being returned to the user. 

Example task:
```yams
- debug:
    var: resp.data
```

Outputs:
```
    "resp.data": {
        "droplet_limit": 100,
        "email": "sammy@digitalocean.com",
        "email_verified": true,
        "floating_ip_limit": 20,
        "status": "active",
        "status_message": "",
        "uuid": "b0ea118ed29548573729203027a0442f9"
    }
```

**Review by BondAnthony — COMMENTED** [review]
LGTM other than the response object.

**Review by BondAnthony — APPROVED** [review]

File: lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+from ansible.module_utils.digital_ocean import DigitalOceanHelper
+from ansible.module_utils._text import to_native
+
+
+def core(module):
+    rest = DigitalOceanHelper(module)
+
+    response = rest.get("account")
+    if response.status_code != 200:
+        module.fail_json(msg="Failed to fetch 'account' facts due to error : %s" % response.json['message'])
```
Comment (pmarques) [inline lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py:68]: Will not be this caught inside the DO helper?

Comment (pmarques) [reply lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py:68]: My fault, only 401 is handled.

**Review by pmarques — APPROVED** [review]

File: lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/digital_ocean/digital_ocean_account_facts.py:46]: `sample` is actual table header in documentation - e.g., [DO Tag](http://docs.ansible.com/ansible/latest/digital_ocean_tag_module.html#return-values) return values table.

**Comment by Akasurde** [comment]
@pmarques @BondAnthony Thanks for reviews.

**Pull request merged**